alexandria-reader/backend
# chore: update package dependencies

## Description

Many of the packages we rely on were outdated, they have all been updated to the latest versions.
These include `jest`, `article-parser`, `dotenv`, `boom`, `uuid` and `ts-node-dev`.
I've added a troubleshooting guide to `README.md`.
I also ran into a few issues, so I've added TODOs in the code for future reference.

## Type of Changes



|     | Type                       |
| --- | -------------------------- |
|     | :bug: Bug fix              |
|     | :sparkles: New feature     |
|     | :hammer: Refactoring       |
|     | :100: Add tests            |
|  x  | :link: Update dependencies |
|  x  | :scroll: Docs              |

## Testing Steps / QA Criteria


Run the app locally and see if anything is broken.
I tested all the basic functionality manually and I didn't have any issues. More extensive testing before merging would be a good idea.
